samples/tfm/tfm_psa_template/README.rst

```diff
@@ -1,13 +1,14 @@
-.. _tfm_cert:
+.. _tfm_psa_template:
+
+TF-M: PSA Template
 
-TF-M: PSA Certified Level 2
 ###########################
```

Comment:
Not an RST expert, but I think the ### needs to be the same length as the title.